src/main/java/org/opensearch/security/securityconf/impl/CType.java

```diff
@@ -27,14 +27,17 @@
 
```

Comment:
This file is cherry-picked these from @willyborankin's recent PR that is out in review.
- [1] #4002